java/ql/src/experimental/Security/CWE/CWE-074/JndiInjectionLib.qll

```diff
@@ -155,13 +159,33 @@ predicate jmxConnectorFactorySinkMethod(Method m, int index) {
   index = 0
 }
 
+/**
+ * Tainted value passed to env `Hashtable` as the prodiver UDL, i.e.
```

Comment:
Typo, "prodiver"

Reply:
Good catch, thanks.
There're 2 typos actually: 'prodiver' and 'UDL' (should be URL) :)